pkg/plugin/plugin.go

```diff
@@ -560,17 +569,58 @@ func CmdDel(args *skel.CmdArgs) error {
 		}
 	}
 
-	// Delete can be called multiple times, so don't return an error if the
-	// device is already removed.
-	err = ns.WithNetNSPath(args.Netns, func(ns.NetNS) error {
-		err = ip.DelLinkByName(args.IfName)
-		if err != nil && err == ip.ErrLinkNotFound {
-			return nil
-		}
+	if netconf.DeviceID != "" {
+		//  SR-IOV Case
```

Comment:
Can we call it multiple times (like with the general case)?

Reply:
sure, added a check to handle `ErrLinkNotFound` error.